### PR TITLE
Updated a few things that seemed out of date in the getting started docs

### PR DESCRIPTION
**Pull request by sjwhitmore**
Was using these myself to remember syntax & a few things seemed like they had changed in recent versions

### CODE REVIEW

File: docs/getting_started/chains.md [hunk 1]
```diff
@@ -12,28 +12,36 @@ This is easy to do with LangChain!
 First lets define the prompt:
 
 ```python
-from langchain.prompts import PromptTemplate
+from langchain import Prompt
```
Comment (hwchase17) [inline docs/getting_started/chains.md:15]: this is PromptTemplate now!

File: docs/getting_started/chains.md [hunk 2]
```diff
@@ -12,28 +12,36 @@ This is easy to do with LangChain!
...
-from langchain.chains import LLMChain
+from langchain import LLMChain
 chain = LLMChain(llm=llm, prompt=prompt)
 ```
 
 Now we can run that can only specifying the product!
 
 ```python
-chain.run("colorful socks")
+chain.predict("colorful socks")
```
Comment (hwchase17) [inline docs/getting_started/chains.md:41]: if predict, i think it needs to be a key word argument

Comment (hwchase17) [reply docs/getting_started/chains.md:41]: both run and predict are valid

**Comment by dev2049** [comment]
stale

**Pull request closed without merging**